src/forklift/core.py

```diff
@@ -233,70 +190,99 @@ def _hash(crate, hash_path):
         #: truncate destination table since we are hashing for the first time
         arcpy.TruncateTable_management(crate.destination)
 
-    shape_token = 'SHAPE@'
-    is_table = _is_table(crate)
+    shape_token = 'SHAPE@WKT'
+    src_id_field = 'src_id'
```

Comment:
No I think it needs to be a new field when added to the temp_table to avoid problems with ESRI managed id fields.